CHANGELOG.md

```diff
@@ -2,12 +2,16 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [Unreleased]
+### Added
+- [#29](https://github.com/Kashoo/synctos/issues/29): Parameter to indicate that an item cannot be modified if it has a value
+
 ## [1.1.0] - 2016-07-15
 ### Added
-- Provide default type filter function ([#26](https://github.com/Kashoo/synctos/issues/26))
+- [#26](https://github.com/Kashoo/synctos/issues/26): Provide default type filter function
 
 ### Fixed
-- Does not return a non-zero exit status when sync function generation fails ([#36](https://github.com/Kashoo/synctos/issues/36))
+- [#36](https://github.com/Kashoo/synctos/issues/36): Does not return a non-zero exit status when sync function generation fails
 
-[Unreleased]: https://github.com/olivierlacan/keep-a-changelog/compare/v1.1.0...HEAD
-[1.1.0]: https://github.com/olivierlacan/keep-a-changelog/compare/v1.0.0...v1.1.0
+[Unreleased]: https://github.com/Kashoo/synctos/compare/v1.1.0...HEAD
+[1.1.0]: https://github.com/Kashoo/synctos/compare/v1.0.0...v1.1.0
```

Comment:
Oops. I accidentally left the [keep-a-changelog](https://github.com/olivierlacan/keep-a-changelog) repo's URL in the original version of the changelog when I copied this as an example. Fixed now.